src/staging/src/templates/staticGeneral1.js

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import { graphql } from "gatsby"
+import SEO from "../components/seo"
+import Header from "../components/header"
+import Menu from "../components/Menu"
```

Comment:
Please use lowercase `menu` in `"../components/Menu"`.  Linux is case sensitive and this will cause build errors

Reply:
Interesting, I think this file was named with an uppercase at some point, regardless I will update.